src/main/java/com/minecolonies/core/colony/buildings/workerbuildings/BuildingKitchen.java

```diff
@@ -88,9 +88,7 @@ public boolean isRecipeCompatible(@NotNull final IGenericRecipe recipe)
             if (isRecipeAllowed.isPresent()) return isRecipeAllowed.get();
 
             final ItemStack output = recipe.getPrimaryOutput();
-            return FoodUtils.EDIBLE.test(output)
-                || FoodUtils.EDIBLE.test(FurnaceRecipes.getInstance()
-                    .getSmeltingResult(output));
+            return FoodUtils.EDIBLE.test(output) || FoodUtils.EDIBLE.test(IMinecoloniesAPI.getInstance().getFurnaceRecipes().getSmeltingResult(output));
```

Comment:
I think it was nicer if we could query this through the class proxy which goes itself to api.getInstance, makes it more readable

Reply:
Can do, but best case I'm making a method in the IFurnaceRecipes interface. There isn't much reason to have this appear in the IMinecoloniesAPI if it's not going to be used and people use the implementation directly.

So either it'll have to use through the IMinecoloniesAPI (either directly like this or through an interface static method), or we rip this thing out of the API completely and just let it be it's own static class. I'm good with either.